src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -146,10 +146,6 @@ private static void createStatus(@NonNull Run<?, ?> build, @NonNull TaskListener
         } else if (Result.FAILURE.equals(result)) {
             statusDescription = StringUtils.defaultIfBlank(buildDescription, "There was a failure building this commit.");
             state = FAILED_STATE;
-        } else if (Result.NOT_BUILT.equals(result)) {
```

Comment:
Wouldn't it be better to comment it out and add a TODO pointing to this PR: https://github.com/jenkinsci/workflow-job-plugin/pull/294